engine/src/test/java/io/zeebe/engine/processor/workflow/deployment/model/validation/ZeebeRuntimeValidationTest.java

```diff
@@ -149,16 +150,9 @@
         // output element expression is not supported
         Bpmn.createExecutableProcess("process")
             .startEvent()
-            .serviceTask(
-                "task",
-                t ->
-                    t.multiInstance(
-                        m ->
-                            m.zeebeInputCollection("foo")
-                                .zeebeOutputCollection("bar")
-                                .zeebeOutputElement(INVALID_PATH_QUERY)))
+            .serviceTask("task", t -> t.zeebeTaskType(INVALID_EXPRESSION))
             .done(),
-        Arrays.asList(expect(ZeebeLoopCharacteristics.class, INVALID_PATH_QUERY_MESSAGE))
+        Arrays.asList(expect(ZeebeTaskDefinition.class, INVALID_EXPRESSION_MESSAGE))
```

Comment:
It seems that you removed the test case for `ZeebeLoopCharacteristics`.

Reply:
Oops